### PR TITLE
docker: add ability to use a custom control socket

### PR DESCRIPTION
**Pull request by iSchluff**
Allows for specifying a custom control socket in the docker entrypoint, which is useful for sharing it with sidecar containers

### CODE REVIEW

**Comment by bradfitz** [comment]
Leaving for @maisem.

File: docs/k8s/run.sh [hunk 1]
```diff
@@ -81,11 +82,11 @@ if [[ ! -z "${TS_EXTRA_ARGS}" ]]; then
 fi
 
 echo "Running tailscale up"
-tailscale --socket=/tmp/tailscaled.sock up ${UP_ARGS}
+tailscale --socket=${TS_SOCKET} up ${UP_ARGS}
```
Comment (maisem) [inline docs/k8s/run.sh:85]: Add quotes here and below

Comment (maisem) [reply docs/k8s/run.sh:85]: Oh I guess you don't need them below. Only here is fine

Comment (iSchluff) [reply docs/k8s/run.sh]: resolved

**Review by maisem — APPROVED** [review]

**Comment by maisem** [comment]
Thanks!

**Pull request merged**